packages/workflow-transformer-async/index.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env node */
+/* eslint-disable global-require */
+
+try {
```

Comment:
I'm starting too see a problem in production with this pattern. 

I wan't to either make `package.main = dist/index`.

Or adapt the pattern in [`workflow-cmd`](https://github.com/havardh/workflow/blob/master/packages/workflow-cmd/index.js) through all packages. Do you have a better suggestion or any preferences?

The production problem came with a missing dependency. This is interpreted as failing to load `dist` and thus tries to load `src/index` while masking the actual error.

Reply:
I have similar thoughts on this, but let's keep it out of this PR.